tests/app/data/test_schema_validation.py

```diff
@@ -0,0 +1,47 @@
+from app.schema_loader import schema_loader
+import json
+import jsonschema
+import logging
+import unittest
+import os
+
+from app import settings
+
+from jsonschema import validate
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestSchemaValidation(unittest.TestCase):
+
+    def test_schema(self):
+
+      errors = []
+
+      for file in schema_loader.available_schemas():
+
+        try:
+
+          schema_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, file), encoding="utf8")
+
+          self.validate_json_against_schema(json.load(schema_file))
+
+        except jsonschema.exceptions.ValidationError as e:
```

Comment:
The try and exception here should be around the `jsonschema.validate` method below.